src/query/functions-v2/tests/it/scalars/mod.rs

```diff
@@ -117,6 +117,21 @@ pub fn run_ast(file: &mut impl Write, text: &str, columns: &[(&str, DataType, Co
                 Value::Column(output_col) => {
                     test_arrow_conversion(&output_col);
 
+                    // Only display the used input columns
+                    let used_columns = expr.column_refs();
+                    let input_domains = input_domains
```

Comment:
some functions should generate result column with row_size even without args.

For example:

`today()`, `random()`

So we should keep at least one column.


And we should refactor in  collect_columns function in https://github.com/datafuselabs/databend/blob/b3c2cb5b8a15e5f5c9c6f10f51edaa0ce162b145/src/query/functions-v2/tests/it/aggregates/mod.rs

Reply:
> some functions should generate result column with row_size even without args.

ok, let's do it in a separate pr. BTW, why `today()` needs the row size either?

Reply:
sorry, `today` does not need.